R/mod_query_data.R

```diff
@@ -161,6 +180,14 @@ mod_query_data_ui <- function(id) {
         )
       )
     ),
+    shiny::fluidRow(
+      column(
+             4,
+             shiny::checkboxGroupInput(ns("providers"), 
+             "Data Source", 
```

Comment:
@JamesBisese - is it possible for the provider filter default setting to be both providers checked? When I am querying data, it appears that if I check one of the boxes, only results from that provider will be included. But if I check neither box, then results from both providers are included. Shouldn't no provider boxes checked mean that no results are returned? This seems like it might be a little confusing for users.